cmd/bosun/expr/expr.go

```diff
@@ -98,6 +99,8 @@ func (e *Expr) ExecuteState(s *State, T miniprofiler.Timer) (r *Results, queries
 	defer errRecover(&err)
 	if T == nil {
 		T = new(miniprofiler.Profile)
+	} else {
```

Comment:
Computations are now only used to show users during testing, never during the scheduled check. Miniprofiler is only present when there's a user sitting at the other side of the request that we will show query and timing stats to. Thus, this is a 100% accurate heuristic for determining if we would like to enable computations.

I initially had it as a separate parameter, but it was always the same as T == nil, so I used that instead so as to not change the calling function signature. Do you have another suggestion? I agree it's a bit weird, but I can't think of anything that would be more useful.